roles/container-engine/containerd/templates/config.toml.j2

```diff
@@ -81,3 +81,6 @@ oom_score = {{ containerd_oom_score }}
 {% if containerd_extra_args is defined %}
 {{ containerd_extra_args }}
 {% endif %}
+
+[plugins."io.containerd.nri.v1.nri"]
```

Comment:
HI @fmuyassarov 

Is it better to add before the `containerd_extra_args`

Reply:
Good catch. Let me fix it quickly.